test/Microsoft.ApplicationInsights.AspNet.Tests/Extensions/HttpRequestExtensionsTest.cs

```diff
@@ -2,7 +2,7 @@
 {
     using Microsoft.ApplicationInsights.AspNet.Extensions;
     using Microsoft.AspNet.Http;
-    using Microsoft.AspNet.Http.Core;
+    using Microsoft.AspNet.Http.Internal;
```

Comment:
DefaultHttpContext used in the test cases is from this namespace. this should be fine IMO